manifests/templates/release/olm/cdi-csv-preconditions.yaml.in

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    cdi.kubevirt.io: ""
+  name: {{.Namespace}}
+{{index .GeneratedManifests "cdi-crd.yaml"}}
```

Comment:
@annastopel can you explain why we need this manifest and when/how it is applied?  Near as I can tell it is not part of OLM and has to be installed out of band by an admin.

Reply:
OLM overview:
OLM i combined of 2 operators: OLM operator and Catalo source operator.
OLM operator :
Consumed csv manifest and installs the operator withrespect to installPlan
provided in csv. It assumes that all of the resources declared in csv are
installed
Catalog operator:
The Catalog Operator is responsible for resolving and installing
ClusterServiceVersions and the required resources they specify. It is also
responsible for watching catalog sources for updates to packages in
channels, and upgrading them (optionally automatically) to the latest
available versions.

So  if we want to test csv manifest only without catalog operator we need
to install manually all the resources it declares.

Preconditions manifest contains all the required info.

Do you think I should describe it in md file? In our md file I put
reference to kubevirt md where they describe the flow (cdi and kubevirt Olm
are very similar)